### PR TITLE
Support Timelike with up to 2 ekos

### PR DESCRIPTION
**Pull request by giacomomagni**
Support time-like evolution with up to 2 ekos.

### CODE REVIEW

**Comment by giacomomagni** [comment]
@t7phy I'll leave to you the pleasure to test and check if it works... 
Most likely there is some corner case left 😅

File: src/pineko/evolve.py [hunk 1]
```diff
@@ -245,10 +245,6 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     kv = pineappl_grid.key_values()
     conv_type_1, conv_type_2 = get_grid_convolution_type(kv)
 
-    # fragmentation function grid?
-    if "timelike" in kv:
```
Comment (giacomomagni) [inline src/pineko/evolve.py:241]: Actually the previous implementation was not even working cause the "True" business

**Comment by scarlehoff** [comment]
Deprecated by #206
